### ISSUE TITLE
macOs Sierra CPU slammed at 100%

### ISSUE THREAD

**Issue by allixsenos** [rsms/fb-mac-messenger#321]
upgraded from 10.11 to 10.12, immediately noticed CPU hogging by Messenger

happens almost immediately after launch, happens on every launch, doesn't seem to matter which chat is open or anything

**Comment by rsms** [comment]
Yeah, I'm sorry about this. I've spent many hours trying to understand where main.js causes these issues. It's a complex interaction with messenger.com, WebView (not WKWebView) and the code this app injects.

I've got a branch of this app that uses Chromium Embedded Framework instead of Apple WebKit which should be a lot more stable, efficient and smooth. But I don't have much time to work on this at the moment.

If there's any help, it should only eat buckets of CPU during launch, but not after it's "started" (when threads and messages are shown.) At least that's what I've been observing on my various macs on 10.9, 10.10, 10.11 and 10.12.
